lib/script.js

```diff
@@ -18,14 +18,14 @@ if (window.history) {
     el.onclick = function (e) {
       e.preventDefault()
       var parser = document.createElement('a')
-      parser.href = el.href
-      var matchedTpl = _getTemplate(el.pathname)
+      parser.href = e.target.pathname
+      var matchedTpl = _getTemplate(e.target.pathname)
       if (matchedTpl) {
         document.write(matchedTpl)
         document.close()
-        history.pushState({}, '', el.href)
+        history.pushState({}, '', e.target)
       } else {
-        window.location = el.href
+        window.location = e.target
```

Comment:
cool ! :metal: 

Regarding technical limits of webpack ~ pushstate plugin cooperation : 
When I create a new page, it isn't available in script.js automatically.
Work around i found is to clear _cache manually and compile again. 

Problem seems to reside in util.addFilesAsWebpackEntries call. 
Do you think it is related ?

If cleaning the _cache is the only way to get sorted, maybe this hint should be explicited in readme ?

See you !